docs/user/example-cases/harvey-flooding/critical-outflow-bc/perlmutter.critical_outflow_bc.batch.base

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#SBATCH -A PLACEHOLDER_PROJECT_ID
+#SBATCH -C PLACEHOLDER_CONSTRAINT
+#SBATCH -q debug
+#SBATCH -t 0:30:00
+#SBATCH -N PLACEHOLDER_N
+
+N=PLACEHOLDER_N;
+n=PLACEHOLDER_n
+is_cpu_run=PLACEHOLDER_CPU_RUN
+is_gpu_run=PLACEHOLDER_GPU_RUN
+RAIN_DIR=PLACEHOLDER_RAIN_DIR
+
+# Sets the RDycore directory
+export RDYCORE_DIR=PLACEHOLDER_RDYCORE_DIR
+
+# Inputdeck for RDycore
+export YAML_FILE=Turning_30m.critical_outflow_bc.yaml
+
+# Load appropriate modules
+MACH=PLACEHOLDER_MACHINE_NAME
+source ${RDYCORE_DIR}/config/set_petsc_settings.sh --mach ${MACH} --config 1
+
+# Sets the path to the `rdycore` exe. It assumes that RDycore was build
+# in the <RDYCORE_DIR>/build-${PETSC_ARCH}.
+export RDYCORE_EXE=${RDYCORE_DIR}/build-${PETSC_ARCH}/bin/rdycore
+
+# Set LD_LIBRARY_PATH (not sure if this is needed anymore)
+export LD_LIBRARY_PATH=$PETSC_DIR/$PETSC_ARCH/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=$CRAY_LD_LIBRARY_PATH:$LD_LIBRARY_PATH
```

Comment:
We might be able to remove one or both of these (and it's a nice opportunity to check).

Reply:
I got the following error when I tried to remove these lines

```
/global/cfs/projectdirs/m4267/gbisht/rdycore/build-pm-cpu-hdf5_1_14_3-opt-64bit-gcc-11-2-0-fc2888174f5/bin/rdycore: error while loading shared libraries: libmuparser.so.2: cannot open shared object file: No such file or directory
```